rosidl_runtime_c/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,139 @@
+More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull request have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull requests must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
+
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rosidl_runtime_c/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rosidl_runtime_c/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rosidl_runtime_c/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rosidl_runtime_c/)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`rosidl_runtime_c` does not have any feature documentation and it will need to be added for higher quality levels.
+
+### Public API Documentation [3.ii]
+
+`rosidl_runtime_c` has some public API documentation.
+
+### License [3.iii]
+
+The license for `rosidl_runtime_c` is Apache 2.0, and a summary is in each source file, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the [LICENSE](../LICENSE) file.
+
+There is an automated test which runs a linter that ensures each file has a license statement.
+
+### Copyright Statements [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `rosidl_runtime_c`.
+
+There is an automated test which runs a linter that ensures each file has at least one copyright statement.
+
+Most recent test results can be found [here](https://ci.ros2.org/job/nightly_linux_release/lastBuild/testReport/rosidl_runtime_c/copyright)
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+There are currently no public features undergoing tests.
+
+### Public API Testing [4.ii]
+
+There are currently no tests for the public API.
+
+### Coverage [4.iv]
+
+`rosidl_runtime_c` does not currently track test coverage.
+
+### Performance [4.iv]
+
+`rosidl_runtime_c` does not currently have performance tests.
+
+### Linters and Static Analysis [4.v]
+
+`rosidl_runtime_c` uses and passes all the standard linters and static analysis tools for a C package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters).
+
+Results of the linting tests can be found [here](https://ci.ros2.org/job/nightly_linux_release/lastBuild/testReport/rosidl_runtime_c/).
+
+## Dependencies [5]
+
+### Direct Runtime ROS Dependencies [5.i/5.ii]
+`rosidl_runtime_c` has the following runtime ROS dependencies:
+* `rosidl_typesupport_interface`
```

Comment:
can we describe the dependencies as it's done [here](https://github.com/ros2/rcl/pull/639/files#diff-5738374578ac1a78c061cd6f5b8dc8f4R143)?

Reply:
Added.